src/nagendra/logicalPrograms/VerifyStringPalindrom.java

```diff
@@ -0,0 +1,30 @@
+package nagendra.logicalPrograms;
+
+/*
+Program 1:  verify string is palindrom or not.
+input : naman
+output : naman is a palindrom string.
+*/
+
+public class VerifyStringPalindrom {
+	
+	void verifyPalindrom(String userInput) {
+		boolean flag = true;
+		for (int i = 0, j = userInput.length() - 1; i < j; i++, j--) {
+			if (userInput.charAt(i) != userInput.charAt(j))
+				flag = false;
```

Comment:
once you update flag then you should put break right, as you know String is not more palindrome